smartcards/card.py

```diff
@@ -133,9 +146,22 @@ def read_long_value(self):
             read_so_far += self.CHUNK_SIZE
             chunk_num += 1
 
-        compressed_content = full_bytes[8 +
-                                        self.short_value_length:total_expected_length]
-        return gzip.decompress(compressed_content)
+        raw_content = full_bytes[40 +
```

Comment:
maybe split this as `8 + 32` for clarity. (I learned this from @beausmith.)

Reply:
Yeah, I did that elsewhere and forgot this spot. Fixed.